packages/clay-charts/src/AreaLineChart.soy

```diff
@@ -4,5 +4,26 @@
  * This renders the component's whole content.
  */
 {template .render}
-	<div class="metal-chart"></div>
+	{@param? loading: bool}
+
+	<div class="metal-chart-container">
+
+		{let $chartAttrs kind="attributes"}
+			{if $loading}
+				hidden
+			{/if}
+		{/let}
+
+		<div class="metal-chart" ref="chart" {$chartAttrs}></div>
+
+		{let $placeholderAttrs kind="attributes"}
+			{if not $loading}
+				hidden
+			{/if}
+		{/let}
+
+		<div ref="placeholder" {$placeholderAttrs}>
+			{call Chart.loading data="all" /}
+		</div>
+	</div>
```

Comment:
Hey @julien, I think it should be possible to pull this markup out into a partial of some kind. It would be nice to not need to duplicate.

Reply:
@Robert-Frampton of course. I'll do that.